### ISSUE TITLE
Promise rejection warning in config/express.js

### ISSUE THREAD

**Issue by tbushman** [freeCodeCamp/hackathon-leaderboard#60]
`Warning: a promise was created in a handler at /Users/traceybushman/Documents/FreeCodeCamp/hackathon-leaderboard/config/express.js:54:3 but was not returned from it, see http://goo.gl/rRqMUw`  
This is probably because the `next()` callback doesn't have anywhere to route to. The sessions handler probably needs to go in `index.route.js`. right after `const router = express.Router();`

**Comment by scissorsneedfoodtoo** [comment]
Yes, I noticed that. I'm afk for the next 5 or 6 hours, so go ahead and merge it if it solves the issue for you!

**Comment by tbushman** [comment]
Didn't fix. :(
